lib/features/reps_core/reps_admin/reps_admin.install

```diff
@@ -36,3 +36,41 @@ function reps_admin_update_7001() {
   module_disable($modules, TRUE);
   drupal_uninstall_modules($modules);
 }
+
+/**
+ * Attach site admin and user manager roles to users maintenance.
+ */
+function reps_admin_update_7002() {
+  features_revert(array(
+    'reps_core' => array(
+      'user_role',
+      'user_permission',
+    ),
+  ));
+  $usernames = array(
+    'danjoju',
+    'kieleda',
+    'lemaith',
+    'libetho',
+    'n0023soa',
+    'settemi',
+    'siebean'
+  );
+  foreach($usernames as $username) {
+    $user = user_load_by_name($username);
```

Comment:
not sure all these users exist in play/prod... you'd better create them if not and check if you do not need to update the authmap table at the same time....